mobile/src/bundles/common/enums/styles/font-family.enum.ts

```diff
@@ -0,0 +1,13 @@
+const FontFamily = {
+    INTER_100: 'inter-thin',
+    INTER_200: 'inter-extralight',
+    INTER_300: 'inter-light',
+    INTER_400: 'inter-regular',
+    INTER_500: 'inter-medium',
+    INTER_600: 'inter-semibold',
+    INTER_700: 'inter-bold',
+    INTER_800: 'inter-extrabold',
+    INTER_900: 'inter-black',
```

Comment:
Let's use just font family names without this enum

Reply:
required constants due to an error: "using the name more than 3 times" [mobile/src/bundles/common/styles/fonts.ts](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/63/files/b00bd81148f2f63354f96e65c3705f6b5458c1b8#diff-42f48890636dacda6b7d42ef55aeae29014f6556736587dbb2bf46be166deb48) . Because of this, I made an enum

Reply:
Ok, bet let's name it like font names "INTER_THIN" etc.